app/helpers/api_page_helper.rb

```diff
@@ -0,0 +1,50 @@
+module ApiPageHelper
+  # ask for current_weather
+  def get_weather(city = 'berlin', api_id)
+    # Rails.cache.clear
+    if api_id == 1
```

Comment:
How come the `api_id` decide what key you need to read the data from? The new object you'll be building for dealing with the weather API should have logic for retrieving the right cached information based on what you're trying to get.

For example, if you want the forecast for Berlin, your cache key should be specific enough so that this new object knows what information to fetch, based on what city you pass into it.